### ISSUE TITLE
Proper manifest file type

### ISSUE THREAD

**Issue by TwitchBronBron** [rokucommunity/brighterscript#472]
Once the file api (#387) changes land, we need to add a `ManifestFile` type that will be loaded into the program and transformable by plugins, as well as dynamically refreshed during watch mode.

**Comment by TwitchBronBron** [comment]
Closing in favor of #1026
